docs/conf.py

```diff
@@ -43,7 +43,8 @@
 modindex_common_prefix = ["quantum_serverless_project."]
 
 # html theme options
-html_theme = "qiskit"
+html_theme = "qiskit-ecosystem"
+html_title = f"{project} {release}"
```

Comment:
This default to `f"{project} {release} documentation"`, which is redundant and noisy. The value gets used for the left sidebar & page title in the browser.